### PR TITLE
Update hosted branch from latest master

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: 03.Client-installation/04.Inventory/docs.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ example of such a script, gathering information on the localization from the net
 #
 # The example script collects geo localization information
 #
-function err() {
+err() {
```
Comment (TheMeaningfulEngineer) [inline 03.Client-installation/04.Inventory/docs.md:73]: @lluiscampos 
Am I missing something here?
I would expect to see the old change in the `hosted` docs, and the new change in `development`.
What I'm seeing is that this is already rendered on hosted, where is the change coming form then? 

image

Comment (TheMeaningfulEngineer) [reply 03.Client-installation/04.Inventory/docs.md:73]: It doesn't say `function err() {` in `hosted`.

File: 05.System-updates-Yocto-Project/05.Customize-Mender/docs.md [hunk 2]
```diff
@@ -246,7 +246,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/<DIRECTORY-WITH-MENDER-CONNECT-CONF>"
 SRC_URI_append = " file://mender-connect.conf"
 
 do_install_append() {
-    install -m 600 ${WORKDIR}/mender-connect.conf ${datadir}/mender/mender-connect.conf
+    install -m 600 ${WORKDIR}/mender-connect.conf ${D}${sysconfdir}/mender/mender-connect.conf
```
Comment (TheMeaningfulEngineer) [inline 05.System-updates-Yocto-Project/05.Customize-Mender/docs.md:249]: @kacf 
Was this you?
(would be cool to see a `blame` equivalent with reviewing :)).

Just hoping to understand the motivation, have been messing around with Ds and datadirs this week, just gathering info. :)

File: 03.Client-installation/04.Inventory/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 03.Client-installation/04.Inventory/docs.md:73]: I do see `function err` in hosted...

image

File: 05.System-updates-Yocto-Project/05.Customize-Mender/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 05.System-updates-Yocto-Project/05.Customize-Mender/docs.md:249]: Comes from https://github.com/mendersoftware/mender-docs/pull/1348

Comment (lluiscampos) [reply 05.System-updates-Yocto-Project/05.Customize-Mender/docs.md:249]: `datadir` is `/usr/share` and the conf file is expected in `sysconfdir` (`/etc/`)

File: 03.Client-installation/04.Inventory/docs.md (hunk 1 above)
Comment (TheMeaningfulEngineer) [reply 03.Client-installation/04.Inventory/docs.md:73]: Strange, I see it now also. Can't explain the picture I posted above.
Perhaps some strange UI issue I can't reproduce any more. 
Afaic we can conclude it.

**Review by TheMeaningfulEngineer — APPROVED** [review]
Looks good to me.
I haven't built it locally, so if this was expected pls let me know.

**Pull request merged**